swagger/api.spec.yaml

```
@@ -1753,6 +1753,12 @@ components:
               type: array
               items:
                 $ref: '#/components/schemas/GroupOut'
+            system_profile:
+              description: >-
+                The host system profile object
+              type: object
+              items:
+                $ref: '#/components/schemas/SystemProfile'
```

The `items` field is used for arrays, not objects. If you run the API service locally and look at the swagger docs, you can see that this change is not being reflected in the example response or response schema.
```suggestion
            system_profile:
              $ref: '#/components/schemas/SystemProfile'
```